pkg/sharky/store.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// # lockless sharding
```

Comment:
this isn't `godoc` style. consider creating a `doc.go` file with godoc style package description, otherwise this can be removed.